lib/Doctrine/ruleset.xml

```diff
@@ -457,6 +461,8 @@
             <property name="spacingAfterLast" value="0"/>
         </properties>
     </rule>
+    <!-- Require no space between increment/decrement operator and its variable -->
```

Comment:
"Require no space" can be intepreted as "Do not require a space", let's maybe reword:

```suggestion
    <!-- Require there be no space between increment/decrement operator and its operand -->
```
Also, "variable" => "operand"